policyengine_us/parameters/gov/states/il/tax/income/exemption/personal.yaml

```diff
@@ -2,12 +2,18 @@ description: Illinois provides the following personal exemption allowance amount
 values:
   2021-01-01: 2_375
   2022-01-01: 2_425
+  2025-01-01: 2_050
 metadata:
+  uprating: gov.irs.uprating
   unit: currency-USD
```

Comment:
Is this rounded?

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/72925608/d66e5a6f-58a2-46ff-8521-47e42a5e0c74)
I added cuz it says "cost of living adjustment"?

Reply:
Does subsection d-5 talk about the rounding of such parameters?

Reply:
> Does subsection d-5 talk about the rounding of such parameters?

i think so yes 
![image](https://github.com/PolicyEngine/policyengine-us/assets/72925608/73f6b1e9-6b19-41ec-953a-424bb06d9f37)